rollup-plugins/production-pruning.js

```diff
@@ -63,24 +63,27 @@ const formatMessage = (node) => {
 
 const createErrorDocumentation = (node) => {
   const errorMessage = formatMessage(node);
-  const errorIndex = errors.push(errorMessage);
 
-  const markdownOutput = `
-## Error ${errorIndex}:
+  if (errors.includes(errorMessage)) {
+    return errors.indexOf(errorMessage);
+  } else {
+    const errorIndex = errors.push(errorMessage);
 
+    const markdownOutput = `
```

Comment:
Does this look bad as a single line? I think it would be cleaner if its possible

Reply:
yeah, makes the markdown file look bad.